python/brotlimodule.cc

```diff
@@ -74,38 +74,40 @@ static PyObject* brotli_compress(PyObject *self, PyObject *args) {
   return ret;
 }
 
+int output_callback(void* data, const uint8_t* buf, size_t count) {
+  std::vector<uint8_t> *output = (std::vector<uint8_t> *)data;
+  output->insert(output->end(), buf, buf + count);
+  return (int)count;
```

Comment:
If should be safe because the decoder won't write huge chunks at once. See decode.c where we typically cast from int to size_t.